shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-distsql/shardingsphere-sharding-distsql-parser/src/main/antlr4/imports/sharding/Keyword.g4

```diff
@@ -178,3 +178,7 @@ KEY
 GENERATOR
     : G E N E R A T O R
     ;
+
+GENERATORS
+    : GENERATOR S
```

Comment:
keyword should not quote others, it is recommended to change to `G E N E R A T O R S`